### ISSUE TITLE
More filters: filter out projects and contexts from list output

### ISSUE THREAD

**Issue by artem-fedosov** [ultralist/ultralist#142]
I have `@waiting` `@home` contexts, also a certain projects like `+other` that host unimportant items. These extra items aren't important in certain contexts, say, when I am at work. Filtering out projects/contexts from list output would make the list less noisy. Following project conventions, the syntax could be:
```
todolist l -waiting  # Filters out issues that have "waiting" as project or context
todolist l -@waitin -+other  # Filters out issues with context "waiting" or project "other"
```

**Comment by gammons** [comment]
It's an interesting idea.  We'd need to brainstorm a bit more on how to make it intuitive to use before implementing.

**Comment by gammons** [comment]
This will be resolved with #215
